### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: @amimodel/compileC.m [hunk 1]
```diff
@@ -224,24 +224,26 @@ function compileC(this)
...
+            amiciSourceList = [amiciSourceList, '"', fullfile(this.wrap_path, 'src', [amiciSourceBaseNames{ii}, '.cpp']), '" '];
+        end
         eval(['mex ' DEBUG COPT ...
             ' -c -outdir ' fullfile(this.wrap_path,'src') ' ' ...
-            fullfile(this.wrap_path,'src','amici.cpp') ' ' ...
-            includesstr]);
+            amiciSourceList ' ' includesstr]);
     end
-        
-    if(this.nxtrue == this.nx)
```
Comment (FFroehlich) [inline @amimodel/compileC.m]: did this check disappear on purpose?

File: models/model_dirac/CMakeLists.txt [hunk 2]
```diff
@@ -83,15 +85,16 @@ ${MODEL_DIR}/model_dirac_xBdot.cpp
 ${MODEL_DIR}/model_dirac_xdot.cpp
 ${MODEL_DIR}/model_dirac_y.cpp
 ${MODEL_DIR}/model_dirac_z.cpp
-${MODEL_DIR}/wrapfunctions.cpp
+ ${MODEL_DIR}/wrapfunctions.cpp
```
Comment (FFroehlich) [inline models/model_dirac/CMakeLists.txt]: unneccessary space

File: @amimodel/compileC.m (hunk 1 above)
Comment (dweindl) [inline @amimodel/compileC.m]: Yes, it was checked twice before. Before each compilation mex call. Now compilation is done in a single call.

File: models/model_dirac/CMakeLists.txt (hunk 2 above)
Comment (dweindl) [inline models/model_dirac/CMakeLists.txt]: Fixed in the template, but did not rewrap test models. Done.

File: @amimodel/compileC.m (hunk 1 above)
Comment (FFroehlich) [inline @amimodel/compileC.m]: Okay, should be fine then.

**Pull request merged**